config/packages/kimai.yaml

```diff
@@ -154,3 +154,9 @@ kimai:
         activeUsersYear: { title: stats.userActiveYear, query: users, begin: '01 january this year 00:00:00', end: '31 december this year 23:59:59', icon: user, color: yellow }
         activeUsersTotal: { title: stats.userActiveTotal, query: users, icon: user, color: red }
         activeRecordings: { title: stats.activeRecordings, query: active, icon: duration, color: red }
+
+    defaults:
```

Comment:
Please comment that complete block and also add a one-line documentation what it is - so something like `# default values for various forms`.
If you get the `Configuration` object correct, it will work without these config settings in `kimai.yaml`. Those values should be set by the administrator in `local.yaml`.

After changing the Configuration object and commenting this block, use `bin/console cache:clear` to test the changes.

Reply:
The local overrides work.  Mine show up as GBP, London and UK.

Reply:
Fine for me then